src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/UnicodeCategoryRangesGenerator.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Text.RegularExpressions.Symbolic
 {
 #if DEBUG
     /// <summary>Utility for generating unicode category ranges and corresponing binary decision diagrams.</summary>
-    [ExcludeFromCodeCoverage]
+    [ExcludeFromCodeCoverage(Justification = "Used from tests to generate src data files")]
```

Comment:
```suggestion
    [ExcludeFromCodeCoverage(Justification = "Used to generate src data files")]
```
It's not really from tests, if running the tests doesn't run this  code.

Reply:
> It's not really from tests, if running the tests doesn't run this code.

It does:
https://github.com/dotnet/runtime/blob/bfceef9dbd57c1ae9170949ea7845c48ca1ad2ce/src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexExperiment.cs#L37-L51